TestControllers/AzureController.psm1

```diff
@@ -99,14 +99,10 @@ Class AzureController : TestController
 		}
 		$this.VmUsername = $azureConfig.TestCredentials.LinuxUsername
 		$this.VmPassword = $azureConfig.TestCredentials.LinuxPassword
+		$this.SSHPrivateKey = $azureConfig.TestCredentials.sshPrivateKey
```

Comment:
If ($this.SSHPrivateKey) { Set-Variable -Name password -Value '' -Scope Global -Force}

Reply:
update here https://github.com/LIS/LISAv2/pull/852/files#diff-72ec8dab08cc97d856da1edfd3f6110cR107